evalml/tests/data_checks_tests/test_data_checks.py

```diff
@@ -488,6 +488,8 @@ def __eq__(self, series_2):
 def test_default_data_checks_across_problem_types(problem_type):
     default_data_check_list = DefaultDataChecks._DEFAULT_DATA_CHECK_CLASSES
 
+    if is_clustering(problem_type):
+        return
```

Comment:
Curious if we should do a `pytest.skip()` here rather than an empty return, the primary reason being so that it'd be easier to find and remember this is something we need to return to to fix in the future once we do get clustering incorporated.